Automattic/pocket-casts-android
# Fastlane: Fastfile updates and version method renaming

## Description
This PR includes: 
- Updated the Release Toolkit from `9.3.0` to `9.3.1`
- Updated the versioning method naming. E.g. `next_build_code` to `build_code_next`
- Change the user confirmations from negative to positive logic. E.g.:
`UI.user_error!('Aborted by user request') if !options[:skip_confirm] && !UI.confirm('Do you want to continue?')`

to 

```
unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
  UI.user_error!("Terminating as requested. Don't forget to run the remainder of this automation manually.")
end
```
- Use consistent Fastlane UI syntax everywhere. E.g. `UI.message('Hello, Pocket Casts!')` instead of `UI.message 'Hello, Pocket Casts!'`

## Testing Instructions
To test some of the renamed versioning methods:
1. Run `bundle exec fastlane new_beta_release`
2. The confirmation message should show the following output: 
```
      Current beta version: 7.56-rc-3
      New beta version: 7.56-rc-4

      Current build code: 9188
      New build code: 9189
```
3. Hit `n` so that a new beta is not actually created
